docs/configuration/backends/docker.md

```diff
@@ -110,7 +110,10 @@ To enable constraints see [provider-specific constraints section](/configuration
 # Required
 # Default: "unix:///var/run/docker.sock"
 #
-endpoint = "tcp://127.0.0.1:2375"
+# swarm classic (1.12-)
+# endpoint = "tcp://127.0.0.1:2375"
+# swarm intergated (1.12+)
```

Comment:
WDYT to replace `swarm integrated` by `swarm mode` which is the real name?

Reply:
good idea!